pkg/internal/checkup/pod/multus.go

```diff
@@ -0,0 +1,42 @@
+package pod
+
+import (
+	"encoding/json"
+
+	corev1 "k8s.io/api/core/v1"
+
+	networkv1 "github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/apis/k8s.cni.cncf.io/v1"
+)
+
+func CreateNetworksRequest(networkRequestsMap map[string]*networkv1.NetworkAttachmentDefinition) (string, error) {
```

Comment:
If the `networkRequestsMap` maps MAC address to NetworkAttachmentDefinnition,
Please consider using the net.HWAddr for it, or to rename it to MACToNADMap.

Reply:
On second thought, if the assosiation is already known to the caller, maybe its better to pass in the selectors from the outside.

Reply:
yep..